src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const res = {};
+
+  const estilos = sourceString.split(';').filter(checkEmpty);
+
+  estilos.forEach((estilo) => {
+    const [key, value] = estilo.split(':');
+
+    res[key.trim()] = value.trim();
```

Comment:
The `trim()` method is used correctly to remove any leading or trailing whitespace from the keys and values. This ensures that the resulting object does not contain any extraneous spaces.